### DISCUSSION TITLE
Custom Type Converter Scope

### DISCUSSION THREAD

**Discussion by DisplayL** [LuckyPennySoftware/AutoMapper#4233] [NA]
Hi everyone 👋 ! I’ve one use case in my current code that I’m struggling a bit using AutoMapper. What ways I have, if possible, to have a type converter applied only to a specific ‘CreateMap’? Say, for example, I have two different logics for converting an int to a DateOnly and in different ‘CreateMap’s I want to apply a different custom type converter. Is this achievable in AutoMapper without having two different mapper configurations? Thanks in advance! Have a nice day 👍

**Comment by lbargaoanu** [comment]
https://docs.automapper.org/en/latest/Value-converters.html

**Reply by DisplayL** [reply]
Thank you! I’m actually using those in my code but I end up with many repeated value conversions. I have 3 maps that convert int to dateonly one way and a fourth in another so I end with 3 maps using the same value converter and the fourth a different. If I had something that i coud apply globally to a specific map

**Reply by jbogard** [reply]
Type Converters don't have any kind of filter or scope as they're global. You can use `ForAllPropertyMaps` if you want to apply some filter for a Value Converter.

**Reply by DisplayL** [reply]
My intent was for a specific CreateMap apply a type conversion avoiding to repeat the same value conversion for many ForMember’s

**Reply by lbargaoanu** [reply]
Surely you need some kind of filter :) `ForAllMembers` would also work.
